### PR TITLE
www-dev_next-steps

### PR DESCRIPTION
**Pull request by kennethgillen**
Next steps for www-dev

### CODE REVIEW

**Comment by kennethgillen** [comment]
That 404 handler applies https://github.com/openmicroscopy/www.openmicroscopy.org/pull/84/files#diff-aa8eb2fe477b2d36ecc0f14d6422513f

**Comment by kennethgillen** [comment]
Proof of concept redirect added - requires (at least) some static content (the logo).

**Comment by sbesson** [comment]
Successfully deployed against `infra-testpr.openmicroscopy.org`. To be tested this morning with @pwalczysko

File: ansible/server-state-playbooks/www-dev/templates/www-dev.openmicroscopy.org.conf.j2 [hunk 1]
```diff
@@ -14,8 +14,17 @@ server {
...
                 # as directory, then fall back to displaying a 404.
-                try_files $uri $uri/ /index.html;
+                try_files $uri $uri/ =404;
                 # Uncomment to enable naxsi on this location
                 # include /etc/nginx/naxsi.rules
         }
+
+        location /community {
+                proxy_set_header Host www.openmicroscopy.org;
+                proxy_pass $scheme://134.36.65.239$request_uri/;
```
Comment (manics) [inline ansible/server-state-playbooks/www-dev/templates/www-dev.openmicroscopy.org.conf.j2]: Is the trailing `/` intentional?

Comment (kennethgillen) [inline ansible/server-state-playbooks/www-dev/templates/www-dev.openmicroscopy.org.conf.j2]: It was in the nginx.com docs I was following, so happy to have advice if you think it's wrong.

**Comment by pwalczysko** [comment]
The first round of testing finished, see [column E](https://docs.google.com/spreadsheets/d/1rqZdGtJHsrpBp8tvH9YrqquCJJu0p7yb_-1aBs05rV0/edit#gid=469737557)

**Comment by sbesson** [comment]
Discussed earlier today at the SA meeting, a proposal is to spend some bounded time (<1h) investigating the migration of this playbook to consume the [openmicroscopy.nginx-proxy role](https://github.com/openmicroscopy/ansible-role-nginx-proxy) which seems to offer lots of nice features required in our work.

File: ansible/server-state-playbooks/www-dev/www-dev.yml [hunk 2]
```diff
@@ -38,7 +58,17 @@
...
-    - role: openmicroscopy.nginx
+    - role: openmicroscopy.nginx_proxy
+      tags: nginxconf
+      nginx_proxy_worker_processes: "{{ ((ansible_processor_count * ansible_processor_cores) / 2) |round|int }}"
+      nginx_proxy_ssl: True
+      nginx_proxy_ssl_certificate: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_cert_filename }}"
+      nginx_proxy_ssl_certificate_key: "{{ nginx_ssl_files_path }}/{{ nginx_ssl_key_filename }}"
+      nginx_proxy_http2: True
+      nginx_proxy_force_ssl: False
+      nginx_proxy_hsts_age: 31536000
```
Comment (manics) [inline ansible/server-state-playbooks/www-dev/www-dev.yml]: Do you definitely want HSTS? `Testing HTTP Strict Transport Security with Care` https://www.nginx.com/blog/http-strict-transport-security-hsts-and-nginx/

**Comment by sbesson** [comment]
Discussed with @kennethgillen, merging this playbook as the current state of the testing. Will start working against the issue reported by @pwalczysko.

**Pull request merged**